src/SFTPFile.cpp

```diff
@@ -18,7 +18,7 @@
  *
  */
 
-#include "threads/mutex.h"
+#include "p8-platform/threads/mutex.h"
```

Comment:
Switch this to angle include style, as it is an external header.
```suggestion
#include <p8-platform/threads/mutex.h>
```

Reply:
Done